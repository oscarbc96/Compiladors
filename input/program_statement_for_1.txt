calc off

b := 2

for (a in b..4) do
  123
  123
  123
done
