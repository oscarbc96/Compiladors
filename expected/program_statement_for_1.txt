1: $t01 := 2
2: b := $t01
3: a := b
4: IF a LEI 4 GOTO 6
5: GOTO 14
6: PARAM 123
7: CALL PUTI,1
8: PARAM 123
9: CALL PUTI,1
10: PARAM 123
11: CALL PUTI,1
12: a := a ADDI 1
13: GOTO 4
14: HALT
